iodata/test/test_fchk.py

```diff
@@ -514,3 +515,184 @@ def test_load_nbasis_indep(tmpdir):
+    check_load_dump_consistency(tmpdir, 'li_sp_orbital.wfn')
+
+
+def test_dump_fchk_from_wfn_lih_cation(tmpdir):
+    check_load_dump_consistency(tmpdir, 'lih_cation_uhf.wfn')
+    check_load_dump_consistency(tmpdir, 'lih_cation_rohf.wfn')
+
+
+def test_dump_fchk_from_wfn_cisd_lih_cation(tmpdir):
+    check_load_dump_consistency(tmpdir, 'lih_cation_cisd.wfn')
+
+
+# def test_dump_fchk_from_wfn_fci_lih_cation(tmpdir):
+#     check_load_dump_consistency(tmpdir, 'lih_cation_fci.wfn')
+
+
+# def test_dump_fchk_from_wfn_fci_lif(tmpdir):
+#     check_load_dump_consistency(tmpdir, 'lif_fci.wfn')
+
+
+def test_dump_fchk_from_wfn_h2(tmpdir):
+    check_load_dump_consistency(tmpdir, 'h2_ccpvqz.wfn')
+
+
+def test_dump_fchk_from_wfn_o2(tmpdir):
+    check_load_dump_consistency(tmpdir, 'o2_uhf_virtual.wfn')
+    check_load_dump_consistency(tmpdir, 'o2_uhf.wfn')
+
+
+def test_dump_fchk_from_wfn_h2o(tmpdir):
+    check_load_dump_consistency(tmpdir, 'h2o_sto3g.wfn')
+    check_load_dump_consistency(tmpdir, 'h2o_sto3g_decontracted.wfn')
+
+
+def test_dump_fchk_from_wfn_ch3(tmpdir):
+    check_load_dump_consistency(tmpdir, 'ch3_rohf_sto3g_g03.fchk')
+
+
+def test_dump_fchk_from_wfn_cah110(tmpdir):
+    check_load_dump_consistency(tmpdir, 'cah110_hf_sto3g_g09.wfn')
+
+
+def test_dump_fchk_from_wfx_h2(tmpdir):
+    check_load_dump_consistency(tmpdir, 'h2_ub3lyp_ccpvtz.wfx')
+
+
+def test_dump_fchk_from_wfx_water(tmpdir):
+    check_load_dump_consistency(tmpdir, 'water_sto3g_hf.wfx')
+
+
+def test_dump_fchk_from_wfx_lih_cation(tmpdir):
+    check_load_dump_consistency(tmpdir, 'lih_cation_uhf.wfx')
+    check_load_dump_consistency(tmpdir, 'lih_cation_rohf.wfx')
+
+
+# def test_dump_fchk_from_wfx_lih_cisd_cation(tmpdir):
+#     check_load_dump_consistency(tmpdir, 'lih_cation_cisd.wfx')
+
+
+# def test_dump_fchk_from_wfx_cah110(tmpdir):
+#     check_load_dump_consistency(tmpdir, 'cah110_hf_sto3g_g09.wfx')
+
+
+def test_dump_fchk_from_molekel_h2(tmpdir):
+    check_load_dump_consistency(tmpdir, 'h2_sto3g.mkl')
+
+
+def test_dump_fchk_from_molekel_ethanol(tmpdir):
+    check_load_dump_consistency(tmpdir, 'ethanol.mkl')
+
+
+def test_dump_fchk_from_molekel_li2(tmpdir):
+    check_load_dump_consistency(tmpdir, 'li2.mkl')
+
+
+def test_dump_fchk_rdms_cc_nitrogen(tmpdir):
+    check_load_dump_consistency(tmpdir, "nitrogen-cc.fchk")
+
+
+def test_dump_fchk_rdms_ci_nitrogen(tmpdir):
+    check_load_dump_consistency(tmpdir, "nitrogen-ci.fchk")
+
+
+def test_dump_fchk_rdms_mp2_nitrogen(tmpdir):
+    check_load_dump_consistency(tmpdir, "nitrogen-mp2.fchk")
+
+
+def test_dump_fchk_rdms_mp3_nitrogen(tmpdir):
+    check_load_dump_consistency(tmpdir, "nitrogen-mp3.fchk")
+
+
+def test_dump_many_consistency(tmpdir):
+    trj = load_fchk_trj_helper("peroxide_opt.fchk")
+    # Write in temporary folder and then read it
+    for mol in trj:
+        mol.nelec = 18
+    fn_tmp = os.path.join(tmpdir, 'test')
+    dump_many(trj, fn_tmp, fmt='fchk')
```

Comment:
In line with the comment above on `dump_many`, I would also suggest to remove this test.